sycl/include/CL/sycl/types.hpp

```diff
@@ -309,34 +307,31 @@ using Rtp = detail::bool_constant<Mode == rounding_mode::rtp>;
 template <rounding_mode Mode>
 using Rtn = detail::bool_constant<Mode == rounding_mode::rtn>;
 
-// signed to signed
-#define __SYCL_GENERATE_CONVERT_IMPL(DestType)                                 \
-  template <typename T, typename R, rounding_mode roundingMode>                \
+// convert signed and unsigned types with an equal size and diff names
+#define __SYCL_GENERATE_CONVERT_IMPL()                                 \
```

Comment:
Why do you need this as macro? The main idea was to auto-generate some code by using pre-processor features. Now you have no arguments to this macro and call it only once - in that case having a macro just doesn't make sense

Reply:
Ok